docs/modules/servers/pages/postgresql/run/run-java.adoc

```diff
@@ -0,0 +1,108 @@
+= Postgresql James Server &mdash; Run
+:navtitle: Run
+
+== Building
+
+=== Requirements
+
+* Java 11 SDK
```

Comment:
should be Java 21 now

Reply:
Hmmm true. 

Maybe we should start taking a serious look into doc versioning as well. Cause postgres app will defo not be available for 3.8.x james version (which is the current version of the doc)

But yes make sense for postgres app to mention java 21